CONTRIBUTING.md

```diff
@@ -1,6 +1,52 @@
 This is a repo to design a beta version of 18f.gsa.gov based on the U.S.
 Web Design Standards and with a complete refresh of much of the content.
 
+## Front end architecture
+### CSS
+- Styling will be built from the US Web Design Standards.
+- CSS methodology will be inherited from the WDS, which inherits mostly from the [18f front end guide](https://pages.18f.gov/frontend/css-coding-styleguide/architecture/).
```

Comment:
Consider WDS --> Draft Standards (this is how we've formally shortened it in the past)